modules/installation/proc-install-rhdh-ocp-helm-gui.adoc

```diff
@@ -1,8 +1,10 @@
-[id='proc-install-rhdh-ocp-helm_{context}']
+// Module included in the following assemblies:
+// assembly-install-rhdh-ocp-helm.adoc
 
-= Deploying {product} on {ocp-short} using Helm Chart
+[id='proc-install-rhdh-ocp-helm-gui_{context}']
+= Deploying {product-short} from the {ocp-short} web console with the Helm Chart
```

Comment:
Another reviewer recently suggested that we remove the product name from the headings altogether, which I disagreed with for SEO and search reasons. However, I think it is fine to use the full product name at the title / assembly level and to use the short product name at the module level. One reason we currently have some modules using the full product name is because they do not currently belong to an assembly, although I believe they should. 

If there is guidance on using long product names in all headings without exception, please share. I also think we should reach a consensus and establish a standard for product names in headings in the future, since every team member currently has a different opinion on the topic.